pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/OwnershipCache.java

```diff
@@ -177,7 +177,7 @@ public boolean checkOwnership(NamespaceBundle bundle) {
     }
 
     /**
-     * Method to get the current owner of the <code>ServiceUnit</code> or set the local broker as the owner if absent.
+     * Method to get the current owner of the <code>NamespaceBundle</code> or set the local broker as the owner if absent.
```

Comment:
```suggestion
     * Method to get the current owner of the <code>NamespaceBundle</code> 
     * or set the local broker as the owner if absent.
```
line is more than 120

Reply:
@pkumar-singh could you please take a look:)